configuration/exodus-pipeline.yaml

```diff
@@ -236,7 +252,7 @@ Resources:
                 Capabilities: CAPABILITY_AUTO_EXPAND,CAPABILITY_NAMED_IAM
                 ParameterOverrides:
                   !Sub '{"env": "${env}", "oai": "${oai}", "project": "${project}"}'
```

Comment:
exodus-lambda-deploy.yaml needs the lambdafunctionrole as well, meaning it will also need to be given to exodus-pipeline.yaml.